dashboard/utils_test.py

```diff
@@ -645,6 +653,43 @@ def test_count_courses_passed_fa(self):
         )
         assert mmtrack.count_courses_passed() == 1
 
+    @override_settings(FEATURES={"FINAL_GRADE_ALGORITHM": "v1"})
+    def test_count_courses_passed_normal(self):
```

Comment:
I marked as `v0` the ones that should be removed later when we make `v1` default